docs/category/category.yaml

```
@@ -361,3 +361,39 @@ paths:
                 status: 401
                 code: UNAUTHORIZED
                 message: The requested URL requires user authorization.
+  /categories/subjects/names:
+    get:
+      security:
+        - cookieAuth: []
+      tags:
+        - Categories
+      summary: Find all subjects's names
+      description: Finds and returns all subjects's names.
```

Describe it more detailed, please
```suggestion
      description: Finds and returns names of all subjects in all categories.
```